client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -1318,7 +1353,8 @@ public void testInvalidValidateQuery() throws IOException{
         assertFalse(response.isValid());
     }
 
-    public void testCRUDIndexTemplate() throws Exception {
+    // Tests the deprecated form of the API that returns templates with doc types (using the server-side's GetIndexTemplateResponse)
+    public void testCRUDIndexTemplateTyped() throws Exception {
```

Comment:
Small comment, but we've been using the suffix `WithTypes` everywhere, instead of `Typed`.